client/components/deposits-overview/index.tsx

```diff
@@ -2,31 +2,63 @@
  * External dependencies
  */
 import * as React from 'react';
-import { Card, CardHeader } from '@wordpress/components';
+import { Card, CardHeader, ExternalLink } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { createInterpolateElement } from '@wordpress/element';
+import NoticeOutlineIcon from 'gridicons/dist/notice-outline';
 
 /**
  * Internal dependencies.
  */
 import { useAllDepositsOverviews } from 'wcpay/data';
 import strings from './strings';
 import NextDepositDetails from './next-deposit';
+import BannerNotice from 'wcpay/components/banner-notice';
 import DepositsOverviewFooter from './footer';
+import './style.scss';
```

Comment:
I move the CSS import here because I think it makes more sense as the CSS applies to the broader `DepositsOverview` component, not just for the `NextDepositDetails`.